templates/default/sql/dml/0.1.0.sql.erb

```diff
@@ -72,30 +67,8 @@ REPLACE INTO variables(`id`, `value`) VALUES
 
 REPLACE INTO variables(`id`, `value`) VALUES ("max_num_proj_per_user", "<%= node['hopsworks']['max_num_proj_per_user'] %>");
 
-REPLACE INTO variables(`id`, `value`) VALUES ("spark_history_server_ip", "<%= @spark_history_server_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("oozie_ip", "<%= @oozie_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("jhs_ip", "<%= @jhs_ip %>");
-
 REPLACE INTO variables(`id`, `value`) VALUES ("hadoop_version", "<%= @hops_version %>");
 
-REPLACE INTO variables(`id`, `value`) VALUES ("rm_ip", "<%= @rm_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("rm_port", "<%= @rm_port %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("epipe_ip", "<%= @epipe_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("livy_ip", "<%= @livy_ip %>");
-
-REPLACE INTO variables(`id`, `value`) VALUES ("yarn_ui_ip", "<%= @yarn_ui_ip %>");
```

Comment:
not urgent, but we should also clean up this on in ec2-init as well 
https://github.com/logicalclocks/ec2-init/blob/master/ec2init/plans/preinit/pre_inits.py#L387